### ISSUE TITLE
using v8.beta-1 undefined method `thread_mattr_accessor' error

### ISSUE THREAD

**Issue by adrienpoly** [hotwired/turbo-rails#527]
While I was testing the beta I encountered an issue between turbo-rails and Rails generators 

In a vanilla Rails 7.1 app when I run a scaffold 
```bash
rails g scaffold post name:string
```

I have this error message
```
/Users/adrien/.asdf/installs/ruby/3.2.2/lib/ruby/gems/3.2.0/gems/turbo-rails-2.0.0.pre.beta.1/lib/turbo-rails.rb:8:in `<module:Turbo>': undefined method `thread_mattr_accessor' for Turbo:Module (NoMethodError)

  thread_mattr_accessor :current_request_id
```

**Comment by adrienpoly** [comment]
I realized that this error occurs also when I start my Rails app. Looks like https://github.com/hotwired/turbo-rails/pull/519 is proposing a fix to this issue

**Comment by adrienpoly** [comment]
as a temporary fix 

adding this to my application.rb solved the error
```
require "active_support/core_ext/module/attribute_accessors_per_thread"
```

**Comment by adrienpoly** [comment]
I check the demo provided by @jorgemanrubia. In the demo repo this error is not present. The difference with my app is that I use Propshaft. If I replace Sprockets by Propshaft in https://github.com/basecamp/turbo-8-morphing-demo then there is the same error. Hope this can help to fix

**Comment by 1klap** [comment]
+1 encountered the exact same issue with propshaft

**Comment by adrienpoly** [comment]
actually this is already solved on main branch with #523
